README.md

```diff
@@ -25,25 +25,109 @@ This is an application which uses Node.js to connect to IBM Db2 Warehouse on Clo
 git clone https://github.com/IBM/crud-using-nodejs-and-db2.git
 ```
 
-### 2. Create IBM Db2 Warehouse on Cloud
+### 2a. Create IBM Db2 Warehouse on Cloud
 
```

Comment:
This doesn't match steps. You need to add 2a, 2b, 3a, 3b in steps link it to each section